src/client/src/shell/routes/SpotRoute.tsx

```diff
@@ -13,10 +14,14 @@ const SpotTileStyle = styled.div`
   margin: auto;
 `
 
-export default ({ match }: RouteComponentProps<{ symbol: string }>) => (
-  <RouteWrapper>
-    <SpotTileStyle>
-      <SpotTileContainer id={match.params.symbol} />
-    </SpotTileStyle>
-  </RouteWrapper>
-)
+export default ({ location, match }: RouteComponentProps<{ symbol: string }>) => {
+  const queryString = location.search.split('?')[1]
```

Comment:
Here I would go with creating some utility method for parsing the query string that includes checks for null etc. There are tried and tested solutions out there you can copy or take inspiration from

Reply:
Thanks